scripts/usb_reset.py

```diff
@@ -90,6 +89,53 @@ def get_cg_dir(domid):
     return "/sys/fs/cgroup/devices/qemu-{}".format(domid)
 
 
+def get_ids_path(device):
+    usb_dir = "/var/run/nonpersistent/usb"
+    if not os.path.isdir(usb_dir):
```

Comment:
This check is no longer needed, if the folder is there it will fail with `errno.EEXIST` and will be ignored

Reply:
done.